subgraph/subgraph.yaml

```diff
@@ -76,11 +76,11 @@ dataSources:
       file: ./src/datasources/Dispatcher.ts
   - kind: ethereum
     name: FortaStaking
-    network: matic
+    network: mumbai
     source:
       abi: FortaStaking
-      address: "0xd2863157539b1D11F39ce23fC4834B62082F6874"
-      startBlock: 20187154
+      address: "0x64d5192F03bD98dB1De2AA8B4abAC5419eaC32CE"
```

Comment:
these addresses should also be updated in `networks.json`

Reply:
Good catch!